app/controllers/hiring_staff/sign_in/dfe/sessions_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
   rescue Authorisation::ExternalServerError => error
     Rollbar.log(:error, error)
     respond_to do |format|
```

Comment:
Does re-raising the exception have the same effect without having a "respond_to" ?